tests/users/test_api_list_users.py

```diff
@@ -179,6 +179,16 @@ def test_list_users_api_with_a_partial_page_and_per_page_query_resource_auth(sel
         self.assertEqual(200, actual_response.status_code)
         self.assertEqual(expected_response, json.loads(actual_response.data))
 
+    def test_list_users_api_with_is_verified_query_resource_auth(self):
```

Comment:
Can you please add 2 more tests, for:
- `is_verified` == false
- `is_verified` == None (i.e.: not being given in the query parameter)